components/ExpandedChapter.tsx

```diff
@@ -41,7 +42,8 @@ const ExpandedChapter = ({
         color={selected ? "PrimaryBlue" : "Grey300"}
         sx={{
           width: "100%",
-          height: { xs: "48px", sm: "64px" },
+          height: { xs: "48px", sm: "64px", md: 'fit-content' },
```

Comment:
| Before | After |
|--------|--------|
| <img width="943" alt="image" src="https://github.com/user-attachments/assets/f3001de2-13fa-4b8c-8f11-b1ba910cbb40"> | <img width="985" alt="image" src="https://github.com/user-attachments/assets/6d8a4a7e-495e-4906-864a-a4b1237882e1">
 |

Reply:
For screen sizes that are just above a tablet, the text on the buttons wasn't wrapping. 

@charcarr what do you think? Should we wrap the text or maybe just have the left buttons and the video stack at a wider screen width?

Reply:
💯 agree with your choice to wrap text